.travis.yml

```diff
@@ -39,8 +45,8 @@ jobs:
       script:
         - make vcctl
         - make images
-        - make e2e-test-job
-    - stage: E2E Scheduling
+        - travis_retry make e2e-test-jobp
```

Comment:
`travis_retry` what does this mean

Reply:
If make e2e-test-jobp failed, it will retry for 3 times.

Reply:
It is a small probability if kind create cluster will failed of write configuration, and this command will retry.